### DISCUSSION TITLE
Can somebody explain me the difference  between directory src/ src2 in front-end part of insights??

### DISCUSSION THREAD

**Discussion by vineyrawat** [frappe/insights#409] [Q&A] [answered]
I'm trying to make some customizations in insights frontend charts, but I'm bit confused between src and src2 can somebody explain me the difference between these two directories.

**Comment by UmakanthKaspa** [comment, marked as answer]
Hi @vineyrawat 

src is for version 2, and src2 is for version 3 of the frontend. Make sure to check which version you're working on before customizing.

Hope this helps!

**Reply by vineyrawat** [reply]
Thanks @UmakanthKaspa !
